Tests/Tests.Kerberos.NET/Crypto/PKInitOctetString2KeyTests.cs

```diff
@@ -16,12 +16,15 @@ public void String2Key_256_0()
         {
             var key = new byte[256];
 
+            var rnd = new Random(42);
+            rnd.NextBytes(key);
```

Comment:
For me it's easier to debug if there are non-0 values, so I randomized the bytes (with constant seed).